### PR TITLE
Feature/feature improving input output images

### PR DESCRIPTION
**Pull request by AlbertPG**

### CODE REVIEW

**Review by enrichp — APPROVED** [review]

File: app/src/main/java/org/pesmypetcare/mypetcare/utilities/ImageManager.java [hunk 1]
```diff
@@ -85,7 +87,13 @@ public static void writeImage(String relativePath, String fileName, byte[] bytes
         }
 
         byte[] bytes = convertToByteArray(bytesList);
-        input.close();
+        input.close();*/
+
+        String imagePath = PATH + File.separator + relativePath + File.separator + imageName + EXTENSION;
+        FileChannel channel = new FileInputStream(imagePath).getChannel();
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/utilities/ImageManager.java:93]: Codacy Issue found: [Avoid instantiating FileInputStream, FileOutputStream, FileReader, or FileWriter](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682&bid=17378767)

File: app/src/main/java/org/pesmypetcare/mypetcare/utilities/ImageManager.java (hunk 1 above)
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/utilities/ImageManager.java:93]: Codacy Issue found: [Ensure that resources like this FileChannel object are closed after use](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetFragment.java [hunk 2]
```diff
@@ -163,13 +166,24 @@ public static void setPetProfileDrawable(Drawable drawable) {
         petProfileDrawable = drawable;
     }
 
+    public static void setIsDefaultPetImage(boolean isDefaultPetImage) {
```
Comment (enrichp) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetFragment.java:173]: comments

File: app/src/main/java/org/pesmypetcare/mypetcare/services/PetManagerAdapter.java [hunk 3]
```diff
@@ -1,6 +1,7 @@
 package org.pesmypetcare.mypetcare.services;
 
 import android.graphics.Bitmap;
+import android.media.Image;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/services/PetManagerAdapter.java:4]: Codacy Issue found: [Unused import - android.media.Image.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682&bid=17378767)

File: app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePet.java [hunk 4]
```diff
@@ -4,6 +4,7 @@
 import org.pesmypetcare.mypetcare.features.pets.UserIsNotOwnerException;
 import org.pesmypetcare.mypetcare.features.users.User;
 import org.pesmypetcare.mypetcare.services.PetManagerService;
+import org.pesmypetcare.mypetcare.utilities.ImageManager;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePet.java:7]: Codacy Issue found: [Unused import - org.pesmypetcare.mypetcare.utilities.ImageManager.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682)

File: app/src/main/java/org/pesmypetcare/mypetcare/utilities/ImageManager.java [hunk 5]
```diff
@@ -121,4 +129,20 @@ public static void deleteImage(String relativePath, String imageName) {
...
+    public static Bitmap getDefaultPetImage() {
+        return defaultPetImage;
+    }
+
+    /**
+     * Get default bytes for pet image.
+     * @return The bytes of pet image
+     */
+    public static byte[] getDefaultBytesPetImage() {
+        return defaultBytesPetImage;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/utilities/ImageManager.java:146]: Codacy Issue found: [Returning 'defaultBytesPetImage' may expose an internal array.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682)

**Review by enrichp — APPROVED** [review]

**Review by xavier-campos-diaz — APPROVED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682&bid=17378767)

File: app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePet.java (hunk 4 above)
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePet.java:7]: Codacy Issue found: [Unused import - org.pesmypetcare.mypetcare.utilities.ImageManager.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682)

**Pull request merged**